app/analysis/views.py

```diff
@@ -3812,14 +3812,24 @@ def file(db, c):
     company.name
 FROM
     nodes LEFT JOIN node_modes ON nodes.id = node_modes.node_id
-    JOIN company ON nodes.company_id = company.id
+    JOIN company ON company.id = nodes.company_id {}
```

Comment:
Use parameterized queries here instead of format strings.

[See here](https://stackoverflow.com/a/775399)

Reply:
yea, I had an issue auto building that using parameterized strings and allowing the list of secondary companies to be as long as needed all in a single query.

Reply:
https://github.com/ace-ecosystem/ACE/commit/c7f0fec83b9c2c4de264d2a0284afd862606d58e